backend/bamboo/schemas/media.py

```diff
@@ -0,0 +1,21 @@
+from enum import Enum
+
+from apiflask import Schema
+from apiflask.fields import Enum as EnumField
+from apiflask.fields import File as FileField
+from apiflask.validators import FileSize, FileType
+
+
+class MediaContentType(Enum):
+    image = "image"
+    slides = "slides"
+
+
+class MediaSchema(Schema):
+    content_type = EnumField(MediaContentType, required=True)
```

Comment:
Its value is the real mimetype of the file, such as `application/json` and `image/png`, not a predefined list of names.

This can be derived from the file itself, instead of requiring input from user.

The media file type(image/slides) may be still useful though, let's determine it from the content type.

So the upload API doesn't need to offer any other info, just a normal file upload.

Reply:
OK! I misunderstood this field, I will generate it using `mimetypes.guess_type`.